src/migrate.rs

```diff
@@ -32,7 +32,7 @@ pub fn migrate(
     let contract_info = get_contract_version(deps.storage)?;
 
     match contract_info.version.as_str() {
-        "2.2.0" => {
+        "2.2.0" | "2.2.1" => {
```

Comment:
Nice, thanks for updating this. Want to get your thoughts on this: So with those previous changes, mainly in test there was a need to upgrade an already created 2.2.0 contract to 2.2.1, so I had to add logic into the migration code to handle a 2.2.0 -> 2.2.1 contract migration (current state -> current state). Are you going to need to do the same here? You already have a new 2.3.0 SC out in test, correct? Are you going to want to migrate it to 2.3.1? Is it worth having "all" contracts on the exact same version?

Reply:
good point on 2.3.0 -> 2.3.1, yeah there is only 1 contract but I will add it in